src/app/core/services/refiner.service.ts

```
@@ -238,41 +238,41 @@ export class RefinerService {
+          device = 'ANDROID';
+        }
+        (window as typeof window & { _refiner: (eventName: string, payload: IdentifyUserPayload) => void })._refiner(
+          'identifyUser',
+          {
+            id: eou.us.id, // Replace with your user ID
+            email: eou.us.email, // Replace with user Email
+            name: eou.us.full_name, // Replace with user name
+            account: {
+              company_id: eou.ou.org_id,
+              company_name: eou.ou.org_name,
+              region: this.getRegion(homeCurrency) + ' - ' + homeCurrency,
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)